tests/rptest/tests/partition_movement_upgrade_test.py

```diff
@@ -101,8 +102,14 @@ def stop_moving_partitions(self):
 
         self.move_worker.join()
 
-    @ok_to_fail  # https://github.com/redpanda-data/redpanda/issues/5868
-    @cluster(num_nodes=6, log_allow_list=RESTART_LOG_ALLOW_LIST)
+    # Allow unsupported version error log entry for older redpanda versions
+    unsupported_api_version_log_entry = re.compile(
+        "kafka rpc protocol - Error\[applying protocol\] .*Unsupported version \d+ for .*"
+    )
```

Comment:
Thanks for adding this!

I think I've grumbled about it before, but it'd be nice to harmonize all the "expected"/fixed error messages by version at some point, so we can roll out some expected bad logs as we add new releases.

To that end, nit: it might be nice to add a comment about what fixed this log message, and what version range it's expected in.

Reply:
done

Reply:
Thanks!